### ISSUE TITLE
Failed: akka-http-nightly-release-10.1:

### ISSUE THREAD

**Issue by chbatey** [akka/akka-http#3332]
```
[error] java.lang.NoSuchMethodError: sbtassembly.AssemblyPlugin$autoImport$.ShadeRule()Lcom/eed3si9n/jarjarabrams/ShadeRule$;
[error] Use 'last' for the full log.
```

https://jenkins.akka.io:8498/job/akka-http-nightly-release-10.1/AKKA_VERSION=default,Node=akka-http-nightly-node,SCALA_VERSION=2.12.10,jdk=AdoptOpenJDK%208/939/consoleFull

This was on quite a few, but not all, of the versions of this build.

**Comment by jrudolph** [comment]
Probably means that the `release-10.1` branch and master sbt builds are not compatible enough any more.

**Comment by jrudolph** [comment]
Should be fixed with #3334.
